File: python/7 kyu/119.py
# https://www.codewars.com/kata/55ea5304685da2fb40000018/train/python

# Passed

def heggeleggleggo(word):
    vowels = set('aeiou ')
    result = str()
    for char in word:
        result += char
        if char.lower() not in vowels:
            result += "egg"

    return result

output = heggeleggleggo("code here")
print(output)

File: nodejs/7 kyu/119.js
// https://www.codewars.com/kata/58712dfa5c538b6fc7000569/train/javascript

// Passed

function countRedBeads(n) {
  if (n < 2) return 0;
  return (n - 1) * 2;
}

const output = countRedBeads(5);
console.log(output);